packages/edit-site/src/components/navigation-inspector/navigation-menu.js

```diff
@@ -34,6 +36,7 @@ const ALLOWED_BLOCKS = {
 		'core/navigation-link',
 		'core/navigation-submenu',
 	],
+	'core/page-list': [ 'core/page-list-item' ],
```

Comment:
If a block list is now being rendered, you may not need the allowedBlocks hack, as the block list will naturally set that.

Reply:
yes, I'm getting rid of that in https://github.com/WordPress/gutenberg/pull/48689